docs/guides/streaming.md

```diff
@@ -29,6 +29,12 @@ There are three steps to streaming data:
   - [deno][entry_server_deno_tsx]
   - [node][entry_server_node_tsx]
 
+<docs-warning>
+
+Streaming works by inserting script tags into the DOM as deferred promises resolve. If your page includes a [Content Security Policy for scripts](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src), you'll either need to include a nonce in your `Content-Security-Policy` header and on your `<Scripts/>` component in the root route, or weaken your securtiy policy by including `script-src 'self' 'unsafe-inline'` in your `Content-Security-Policy` header.
+
+</docs-warning>
```

Comment:
As discussed in Discord, let's also update this to mention adding the nonce to `renderToPipeableStream` and cross-link to the React docs if they have a section non that

Reply:
👍